.scripts/linux/build-minimal.sh

```diff
@@ -11,6 +11,6 @@ BUILD_CONFIGURATION=${1:-Release}
 
 mkdir -p build
 cd build
-cmake .. -DCMAKE_BUILD_TYPE=${BUILD_CONFIGURATION} -DWARNINGS=Off -DBUILD_JAVA=Off -DBUILD_PYTHON=Off -DBUILD_TESTS=Off
+cmake .. -DCMAKE_BUILD_TYPE=${BUILD_CONFIGURATION} -DBUILD_VALGRIND_TESTS=On -DWARNINGS=Off -DBUILD_JAVA=Off -DBUILD_PYTHON=Off -DBUILD_TESTS=Off
```

Comment:
so how do DBUILD_VALGRIND_TESTS and DBUILD_TESTS relate?

Reply:
removed